bindings/dart/lib/src/rust/api/opendal_api.dart

```diff
@@ -0,0 +1,176 @@
+// This file is automatically generated, so please do not edit it.
+// @generated by `flutter_rust_bridge`@ 2.7.1.
+
+// ignore_for_file: invalid_use_of_internal_member, unused_import, unnecessary_import
+
+import '../frb_generated.dart';
+import 'package:flutter_rust_bridge/flutter_rust_bridge_for_generated.dart';
+
+// Rust type: RustOpaqueMoi<flutter_rust_bridge::for_generated::RustAutoOpaqueInner<Capability>>
+abstract class Capability implements RustOpaqueInterface {
```

Comment:
Please don't expose the capability unless we decide to support it.

Reply:
removed